portal/v2/.eslintrc.yml

```diff
@@ -22,4 +22,6 @@ settings:
     fragment: "Fragment"
     version: "detect"
     flowVersion: "0.53"
-rules: {}
+rules: {
+  no-undef: "off"
```

Comment:
`no-undef` presumably being disabled because of errors with JSX typing global variables. Eslint [recommendation](https://github.com/typescript-eslint/typescript-eslint/blob/main/docs/linting/TROUBLESHOOTING.md#i-get-errors-from-the-no-undef-rule-about-global-variables-not-being-defined-even-though-there-are-no-typescript-errors) is to disable the `no-undef` rule for TypeScript projects because TypeScript's type checker is much more robust.

NIT: That said, this rules section no longer needs the `{}` in the yaml.

Reply:
Yep it was firing this check on some pretty unnecessary stuff so I added this, and fixed!